src/MonoMod.UnitTest/MonoMod.UnitTest.csproj

```diff
@@ -9,16 +9,11 @@
     <Description></Description>
     <PackageTags>$(PackageTags)</PackageTags>
 
-    <TargetFrameworks>net46;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 16.6.0 And $(NETCoreAppMaximumVersion) &gt;= 5.0">$(TargetFrameworks);net5.0</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.0.0 And $(NETCoreAppMaximumVersion) &gt;= 6.0">net6.0;$(TargetFrameworks)</TargetFrameworks>
-    <TargetFrameworks Condition="$(MSBuildVersion) &gt;= 17.4.0 And $(NETCoreAppMaximumVersion) &gt;= 7.0">$(TargetFrameworks);net7.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;net7.0;net46;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
I still think we should keep TFMs in sorted order, whether ascending or descending.

Reply:
I've kept .NET 6.0 as the first entry for the same reason as above, though I've added the same comment present in the `Common.props` file here as well now.